### ISSUE TITLE
Mobile Main Menu can't click top Menu Option when there is a dropdown

### ISSUE THREAD

**Issue by VBGAMER45** [SimpleMachines/SMF#6931]
On a normal desktop clicking on the main menu option for instance "Moderate" takes you to the Moderation Dashboard.

On mobile you can't click that top  menu option and can only click on the submenus. This is true with any other  menu option that has a dropdown such as the Admin and Members options

Tested on iPhone SE version 2

Screenshot
![My Community - Index](https://user-images.githubusercontent.com/978002/129135053-1cd48253-e9e4-46f0-b3e4-89de421d53aa.png)

**Comment by DiegoAndresCortes** [comment]
Does it work if you click/tap twice?

**Comment by VBGAMER45** [comment]
It sometimes works if you click twice. Normally occurs after the 2nd or 3rd try it seems.

**Comment by DiegoAndresCortes** [comment]
I think this one is tricky, anyone want to provide feedback?
I remember adding in some situations the same parent button to the child buttons to allow for accessing the main button.

**Comment by Sesquipedalian** [comment]
> I remember adding in some situations the same parent button to the child buttons to allow for accessing the main button.

I ran into a similar problem when creating a custom theme that changed the hover menus into click menus, and I had to use a similar solution.

**Comment by XinYenFon** [comment]
https://github.com/joeldbirch/superfish/issues/182

**Comment by jdarwood007** [comment]
Anyone able to reproduce this?
Tried on a Pixel with firefox and it works.

**Comment by VBGAMER45** [comment]
Still noticing it on the latest with Sarfari on simplemachines.org with moderate.  And for instance another site https://www.smfhacks.com/demos21/ clicking on the gallery button just shows submenus but clicking main icon doesn't take you the gallery.

**Comment by DiegoAndresCortes** [comment]
Yes can reproduce it too.

I think it's established that the recommended approach for dropdowns is two taps or two clicks to avoid accidental redirections, not sure if we actually want to do something with this.

**Comment by jdarwood007** [comment]
Could we increase the box size for the drop-down arrow and move the tap event to that area?  Taping the main menu item still takes you to its action, while tapping the arrow expands the menu?

Additionally, we could also duplicate the main menu item as a child menu item.

Is this something we can resolve in 3.0 with the new theme changes planned in alpha5?

**Comment by DiegoAndresCortes** [comment]
I like to duplicate the parent in my themes, it's easier and we minimize the chances of custom themes breaking the accessibility of an arrow.

**Comment by jdarwood007** [comment]
Can we get this resolved with #7933, or should we send a separate PR up?

**Comment by Sesquipedalian** [comment]
> Can we get this resolved with #7933, or should we send a separate PR up?

Any fix for this should happen either in #7933 itself or in some PR created after #7933 has been merged.
